tools/clang/lib/SPIRV/DeclResultIdMapper.cpp

```diff
@@ -3459,15 +3459,15 @@ void DeclResultIdMapper::decorateInterpolationMode(
 
 SpirvVariable *DeclResultIdMapper::getBuiltinVar(spv::BuiltIn builtIn,
                                                  QualType type,
-                                                 SourceLocation loc) {
+                                                 SourceLocation loc,
+                                                 spv::StorageClass sc) {
```

Comment:
I like to avoid default parameters when possible. Could we split this into two function. One takes the storage class, and does not include the switch that sets the storage class. The other does not take the storage class. It infers the stroage class and then calls the first function.

Reply:
Sure, I'll push up a change with this in it before looking into allowing the attribute on variables.